modules/editable-layers/src/editable-layers/editable-geojson-layer.ts

```diff
@@ -119,7 +119,7 @@ export type EditableGeoJsonLayerProps<DataT = any> = EditableLayerProps & {
   mode?: any;
   modeConfig?: any;
   selectedFeatureIndexes?: number[];
-  onEdit?: (updatedData?, editType?: string, featureIndexes?: number[], editContext?) => void;
+  onEdit?: (editAction: EditAction<DataT>) => void;
```

Comment:
this is a good catch, surprised it made it all the way into a release like this and wasn't mentioned sooner or caught by a test. Lets try to add a test in here to maintain integrity. 

Something as simple as https://github.com/visgl/deck.gl-community/blob/d0c84e82248cd4e213a15205e7e34bd57ae578e4/modules/editable-layers/test/lib/layers/elevated-edit-handle-layer.spec.ts could do the job.